tests/network/user_defined_network/conftest.py

```
@@ -164,6 +167,35 @@ def udn_pod(
         namespace=udn_namespace.name,
         containers=[{**POD_CONTAINER_SPEC, "name": "udn-container"}],
         client=admin_client,
+        label=ALLOWED_POD_LABEL,
+    ) as pod:
+        pod.wait_for_status(status=Pod.Status.RUNNING)
+        yield pod
+
+
+@pytest.fixture()
```

Ruff (0.16.1)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)